### PR TITLE
Bug 1157440 - More prominent display of start time for upcoming events.

### PR DESCRIPTION
**Pull request by blossomica**
I made a fix for this, I am not sure if It looks the way it was envisioned to look like so If there is a need for any changes here  you can let me know.  I forgot to name the PR "fixes bug ..." but If that's a problem I can fix that. I realized it after seeing the message on github.

### CODE REVIEW

File: airmozilla/main/templates/main/event.html [hunk 1]
```diff
@@ -89,6 +89,21 @@ <h1 class="entry-title">
...
+             &mdash;
+             {{ event.location_time.strftime("%I:%M%p") }}
+              {% if not event.location.name.startswith('Cyberspace') and not 
+                 event.location.name.startswith(settings.DEFAULT_PRERECORDED_LOCATION[0]) %}
+                 in {{ event.location.name }}
+              {% endif %}
+            {% endif %}
+           {% endif %}
+         </p>
+      </h4>
```
Comment (peterbe) [inline airmozilla/main/templates/main/event.html]: If `{% if event.is_upcoming() %}` is false, doesn't that mean you end up with
```html
<h4>
  <p>
  </p>
</h4>
```

**Comment by blossomica** [comment]
@peterbe - Yes you are right. I didn't think of that part but I fixed it, thanks. You can let me know if it is okay now or it needs any other changes.

**Comment by peterbe** [comment]
r+

**Comment by blossomica** [comment]
@peterbe - Do I always have to wait for Travis to finish scanning the last pull request after I get r+ when the difference is so slight that I could just rebase it and then let Travis run a scan on the rebased final file instead?

**Comment by peterbe** [comment]
No, as soon as you get the r+ you can rebase and force push into the branch even if travis hasn't finished the un-squashed version, so to say.

**Comment by blossomica** [comment]
@peterbe - All right, that's good to know. I rebased it.

**Pull request merged**